src/material-examples/cdk-datepicker-formats/cdk-datepicker-formats-example.ts

```diff
@@ -0,0 +1,55 @@
+import {Component} from '@angular/core';
+import {FormControl} from '@angular/forms';
+import {MomentDateAdapter} from '@angular/material-moment-adapter';
+import {DateAdapter, CDK_DATE_FORMATS, CDK_DATE_LOCALE, CalendarView} from '@angular/cdk/datetime';
+
+// Depending on whether rollup is used, moment needs to be imported differently.
+// Since Moment.js doesn't have a default export, we normally need to import using the `* as`
+// syntax. However, rollup creates a synthetic default module and we thus need to import it using
+// the `default as` syntax.
+import * as _moment from 'moment';
+// tslint:disable-next-line:no-duplicate-imports
+import {default as _rollupMoment} from 'moment';
```

Comment:
Does this example need to use `moment` since there's a separate example just highlighting that?

Reply:
We use `moment` to create a custom date format

Reply:
But couldn't you just have one example that both uses moment and the custom format?